src/main/src/main/java/org/geoserver/security/urlchecks/AbstractURLCheck.java

```diff
@@ -43,20 +43,24 @@ public void setDescription(String description) {
         this.description = description;
     }
 
-    /** @return the enable */
+    /** @return the enabled status */
     @Override
-    public boolean isEnabled() {
+    public Boolean isEnabled() {
```

Comment:
This field is not optional, please keep "boolean" if possible. If not stated, the rule should be enabled.
If you're having issues deserializing rules that are missing the value, the internal field could become a Boolean, and then you just check if it's null in the getter, and default to true.

Otherwise please clarify why the API was modified.

Reply:
Reverted.